jupyter-nextflow/nextflow-welcome.html

```diff
@@ -39,21 +39,30 @@ <h1 class=header>Welcome to the Nextflow Workspace</h1>
   </ul>
   <h2 class=header>Get started with Nextflow</h2>
   <p>If you are new to Nextflow, visit <a
-      href=https://www.nextflow.io>nextflow.io</a> for detailed information.
+      href=https://www.nextflow.io>nextflow.io</a> for detailed information. 
+    You can also learn more about using Nextflow in BRH Workspace in the BRH Documentation <a
+      href=https://uc-cdis.github.io/BRH-documentation/01-home/</a>. 
   </p>
   <p>This workspace is set up to run Nextflow workflows in AWS Batch. Your
-    Nextflow configuration must include the Batch queue, IAM role ARN and work
-    directory that were created for you automatically. The sample configuration
+    Nextflow configuration must include the Batch queue (in our config file, 
+    <code>queue</code>), IAM role ARN (<code>jobRole</code>), and work
+    directory (<code>workDir</code>) that were created for you automatically. The sample configuration
     file will allow you to run simple workflows and can be adapted to your
     needs. To get started:
   </p>
   <ul>
-    <li>Open a new "Launcher" tab and open a terminal.
+    <li>Open a new "Launcher" tab by clicking the + next to <code>nextflow-welcome.html</code>
+    <li> Scroll down until you see the option for terminal. Click to open a terminal.
     <li>You can find your sample configuration file in the "data" directory.
       Copy it to a new "nextflow.config" file by running this command in the
       terminal: <code>cp
       ./data/sample-nextflow-config.txt ./nextflow.config</code>
     <li>Launch a Hello World workflow by running this command in the terminal:
       <code>nextflow run hello</code>
   </ul>
+  <p>Since the /pd directory is the only directory that will allow files to persist
```

Comment:
it's more straightforward to navigate to `pd` first, and then follow the instructions, which is what was added in the previous PR here https://github.com/uc-cdis/containers/commit/d34a17af0e38308eb04e3420f797f30dd48126a6

Reply:
I found it hard to understand, and we specifically had feedback from Xiao that said this needed updating. So that's why I added what I added. The current users (me and Xiao) missed that line (unless you just merged it? i'm a little confused)

Reply:
yes it was just added, i sent it this morning https://cdis.slack.com/archives/C0552E6EAM7/p1714405591661169?thread_ts=1714398082.999189&cid=C0552E6EAM7

Reply:
Gotcha. I resolved the merge conflicts with your earlier merge